feat: update dependencies, install angular-material and convert the a… (#18)
xlts-dev/angularjs-webpack-babel: README.md

> npm build

Reviewer: Not directly related to this PR, but shouldn't this be

```suggestion
npm run build
```

```

> For single run unit testing

Reviewer: To be a bit more clear, I might change this to something like:

```suggestion
For running the unit tests once
```

```
npm run test:unit:single
```

> For watch unit testing

Reviewer: To be a bit more clear, I might change this to something like:

```suggestion
For running the unit tests in watch mode
```

```
npm run test:unit:watch
```

> For lint

Reviewer: ```suggestion
For linting
```

```

> npm lint

Reviewer: ```suggestion
npm run lint
```

```

For e2e testing

```
npm run test:e2e
```

> For test the app (lint, unit and e2e testing)

Reviewer: ```suggestion
For running all checks/tests for the app (lint, unit and e2e tests)
```

```

> npm run test

Reviewer: This could also be:

```suggestion
npm test
```
